docs/help/contributor/github.md

```diff
@@ -1,9 +1,4 @@
----
-title: "GitHub workflow for Knative documentation"
-linkTitle: "Using GitHub"
-weight: 10
-type: "docs"
----
+# GitHub workflow for Knative documentation
```

Comment:
looks like some of the syntax in this file doesn't work with mkdocs (specifically the `## Set up your local machine{#clone}` bit - mkdocs doesn't have `{#clone}` to set up an anchor id, instead the id is generated based on the header text (https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages)

Reply:
Thank you; well-spotted.